ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts

```diff
@@ -2500,6 +2502,55 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'DSR Gigaflare',
+      // 6D9A fires first, followed by 6DD2, then 6DD3
+      // 6D99 is cast by boss at the center
+      // Only need to compare the rotation of 6D9A to 6DD2
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: ['6D9A', '6DD2'], source: 'Dragon-king Thordan' }),
+      durationSeconds: 10,
+      infoText: (data, matches, output) => {
+        // Positions are moved up 100 and right 100
+        const x = parseFloat(matches.x) - 100;
+        const y = parseFloat(matches.y) - 100;
+
+        // Collect Gigaflare position
+        matches.id === '6D9A' ? data.firstGigaflare = [x, y] : data.secondGigaflare = [x, y];
+        if (data.firstGigaflare !== undefined && data.secondGigaflare !== undefined) {
+          // Store temporary copies and remove data for next run
+          const first = data.firstGigaflare;
+          const second = data.secondGigaflare;
+          delete data.firstGigaflare;
+          delete data.secondGigaflare;
+
+          if (first[0] === undefined || first[1] === undefined || second[0] === undefined || second[1] === undefined) {
+            console.error(`Gigaflare: missing coordinates`);
+            return;
+          }
+          // Compute atan2 of determinant and dot product to get rotational direction
```

Comment:
!!!

Reply:
> 
> I think this could say "Back left -> Clockwise" sort of thing. They are definitely boss relative.
> 
> Let me know if you have a chance to test this out, as I may or may not have a chance in the next week or two. I'm not sure that I know this math well enough to know if there are edge cases to worry about.

From my checks in typescript sim (pulling x,y from fflogs reports) and raidsim of my own logs it seems to at least be correct. I'll likely be able to do a test in game for how it feels on Sunday.

Here are some more details on how the algorithm works: https://stackoverflow.com/a/16544330 . I think the only way for a 0 value to be returned is if the two coordinates are the same, which shouldn't be possible.

As for getting relative to boss direction for a Back Left => clockwise like call, I think I would need to add tracking of the 6D99 cast to get facing. I know the x,y appear to be accurate, but I haven't really looked at if the facing value there is accurate.